### PR TITLE
[EAM-856] Enhancing Swagger documentation support 

### PR DESCRIPTION
**Pull request by BoykoBorisov**

### CODE REVIEW

File: src/main/java/ch/cern/eam/wshub/core/services/grids/entities/GridRequest.java [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
 import net.datastream.schemas.mp_functions.gridrequest.GRID;
 
 import javax.xml.bind.annotation.XmlElement;
 import javax.xml.bind.annotation.XmlElementWrapper;
 import java.io.Serializable;
 import java.util.*;
 
+@JsonIgnoreProperties(ignoreUnknown = true)
```
Comment (LukaszPater) [inline src/main/java/ch/cern/eam/wshub/core/services/grids/entities/GridRequest.java:12]: I believe we should remove it as it is not the case today and not the case for all remaining entities.

File: pom.xml [hunk 2]
```diff
@@ -118,6 +118,12 @@
             <version>5.6.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-annotations</artifactId>
+            <version>2.11.1</version>
+            <scope>compile</scope>
+        </dependency>
```
Comment (LukaszPater) [inline pom.xml:126]: I assume it's the same version like in wildfly 22?

**Review by LukaszPater — CHANGES_REQUESTED** [review]
Hi Boyko, please check my two comments. Thanks for removing the annotations from the setters ;-).

File: src/main/java/ch/cern/eam/wshub/core/services/grids/entities/GridRequest.java (hunk 1 above)
Comment (BoykoBorisov) [reply src/main/java/ch/cern/eam/wshub/core/services/grids/entities/GridRequest.java]: Was used for ignoring the getRequestKey() method. Now a JsonIgnore annotation is applied to the method directly,

**Pull request merged**